ts/src/binanceus.ts

```diff
@@ -19,6 +19,7 @@ export default class binanceus extends binance {
                 'api': {
                     'web': 'https://www.binance.us',
                     'sapi': 'https://api.binance.us/sapi/v1',
+                    'sapi_v3': 'https://api.binance.us/sapi/v3',
```

Comment:
Hum, is this a new category?

Reply:
seems so, look at the different v1/v3 between these two:
https://docs.binance.us/#get-sub-account-assets
https://docs.binance.us/#get-master-account-39-s-total-usd-value

Reply:
right, binance is not going to use these paths at this moment, however, I've just added with the intention that all endpoints were available in binanceus so users could implicitly call them.

Reply:
at this moment, it is a bit unclear foir me too, how we should make binance.ts (which should be agnostic) to use those unknown paths. however, let's put that step for a bit future, and at this moment, implicit endpoints calls can be supported.